csv-data-management/extract_data_from_csv.py

```diff
@@ -425,6 +451,7 @@ def _df_to_csv(df):
     df.to_csv('/tmp/{}'.format(file_name), encoding='utf-8', header='true',compression="zip")
     filepath = '/tmp/{}'.format(file_name)
     ch_res = create_cyops_attachment(filename=filepath,name=file_name,description='Create by CSV Data Management Connector')
+    remove(filepath)
```

Comment:
can we put the remove in an except block so that file gets removed even if create_attachment fails

Reply:
Done